mshv-bindings/src/lib.rs

```diff
@@ -6,7 +6,8 @@
 #![allow(non_upper_case_globals)]
 #![allow(non_camel_case_types)]
 #![cfg(target_arch = "x86_64")]
-
+#![allow(clippy::too_many_arguments)]
+#![allow(clippy::missing_safety_doc)]
```

Comment:
Is this warning about lacking comments on unsafe blocks? If so, we should write that instead of ignoring this warning.